lego/apps/events/tasks.py

```diff
@@ -577,13 +580,15 @@ def check_that_pool_counters_match_registration_number(self, logger_context=None
     self.setup_logger(logger_context)
 
     events_ids = Event.objects.filter(
-        start_time__gte=timezone.now(), merge_time__gte=timezone.now()
+        Q(start_time__gte=timezone.now()),
```

Comment:
Is this necessary?

Reply:
Well, otherwise we'd be checking all old events? This makes sure we only test for events that are in the future.

Reply:
oh shoot I'm sorry, I read `merge_time__gte` and thought it was a duplicate of the one below ..